core/src/main/java/hudson/node_monitors/ClockMonitor.java

```diff
@@ -53,10 +54,12 @@ public ClockDifference getDifferenceFor(Computer c) {
      */
     @Deprecated
     @Restricted(NoExternalUse.class)
+    @SuppressFBWarnings(value = "MS_PKGPROTECT", justification = "for backward compatibility")
```

Comment:
At some point we should do a pass through things like this and delete any not actually being used, but certainly out of scope here.